.travis.yml

```diff
@@ -2,7 +2,10 @@ sudo: false
 language: java
 jdk: openjdk7
 script: 
-  - mvn deploy site site:deploy
-branches:
-  only:
-    - develop
+  - mvn verify
```

Comment:
This change will build and check the code for all branches. It won't run for this PR because Travis uses the PR target branch's (develop) .travis.yml for security reasons. But it does run on my fork https://travis-ci.com/liujoshua/BridgeJavaSDK/builds/81468431

Things work a bit differently on Windows and I need to hack some things to build, so this is particularly helpful for me to verify my changes work generally.

Deployment still happens on develop.